src/decorations/functions.jl

```diff
@@ -33,6 +33,9 @@ end
 
 in(x::T, a::DecoratedInterval) where T<:Real = in(x, interval_part(a))
 
+@inline Base.:(==)(xx::DecoratedInterval, yy::Interval) = false
```

Comment:
`==` should already be imported, so you can just write `==`. Inlining should be automatic for a simple function like this I guess.